README.md

```diff
@@ -106,6 +106,10 @@ Jobless.cv do
 end
 ```
 
+## TODO
+
+* Add support of the projects, because you can work on the several projects in the same company.
+
 ## Ports
```

Comment:
I'm not sure that would be a good idea. It seems like it would introduce a lot of complexity but would benefit users only in a few edge-case situations. I'd love to keep this tool as simple as possible.